src/main/resources/templates/fragments/inlineYesNoRadioButtons.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+  <title></title>
+</head>
+<body>
+<fieldset class="govuk-fieldset" th:fragment="inlineYesNoRadioButtons">
+
+  <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+    <h1 class="govuk-fieldset__heading">
+      <span th:text=${legendText}></span>
+    </h1>
+  </legend>
+
+  <span class="govuk-error-message"
+        th:id="${radioButtonsFieldName} + '-errorId'"
+        th:if="${#fields.hasErrors('__${radioButtonsFieldName}__')}"
+        th:each="e : ${#fields.errors('__${radioButtonsFieldName}__')}" th:text="${e}">
+            </span>
+
+  <!-- Radio Buttons -->
+  <div class="govuk-radios govuk-radios--inline" data-module="radios">
+
+    <div class="govuk-radios__item">
+      <input class="govuk-radios__input show-text"
```

Comment:
Is there any time where the 'No' selection would reveal the text box? I assume not, but is it worth catering for the scenario by making the class dynamic. This could be done using `th:classappend`

Reply:
After speaking with Brad, we both agreed we will do this change in the future if this is needed.

This fragment is used by accounting policies screens and it will show a text box when `yes` is selected.  we will revisit this If this logic changes(e.g. `no` displays the text);